unittests/files/opt/seagate/sspl/setup/resource_map/lr2/test_storage.py

```diff
@@ -142,6 +143,25 @@ def test_get_drives(self, encl_response):
         assert specifics['storage-pool-name'] == 'poola'
         assert specifics['location'] == '0.0'
 
+    @patch(
+        "files.opt.seagate.sspl.setup.resource_map.lr2.storage.StorageMap.get_realstor_encl_data"
+    )
+    def test_get_fanmodules_info(self, encl_response):
+        storage_map = self.create_storage_obj()
+        encl_response.return_value = json.loads(
+            FANMODULES_RESPONSE
+        )['api-response']['fan-modules']
+        resp = storage_map.get_fanmodules_info()
+        assert resp[0]['uid'] == 'fan_module_0.0'
+        assert resp[0]['health']['status'] == 'OK'
+        assert resp[0]['health']['description'] == 'FAN is in good health'
```

Comment:
Align this description field after addressing above comment.
Then it will be something like:
{uid} is in good health

Reply:
Done.